lib/plugins/package/lib/packageService.js

```diff
@@ -66,15 +67,16 @@ module.exports = {
   },
 
   async packageService() {
-    this.serverless.cli.log('Packaging service...');
+    legacy.log('Packaging service...');
+    progress.get('main').notice('Packaging');
     let shouldPackageService = false;
     const allFunctions = this.serverless.service.getAllFunctions();
     let packagePromises = allFunctions.map(async (functionName) => {
       const functionObject = this.serverless.service.getFunction(functionName);
       if (functionObject.image) return;
       functionObject.package = functionObject.package || {};
       if (functionObject.package.disable) {
-        this.serverless.cli.log(`Packaging disabled for function: "${functionName}"`);
+        legacy.log(`Packaging disabled for function: "${functionName}"`);
```

Comment:
Is there an agreement to not show this message now?

I think we should hide only logs, for which we have counterparts, or for which we agreed they will not be present in new version (otherwise we risk hiding some information unintentionally, it'll be easy to forget about this case in future processs).

So I'd either leave this log not switched, or decide how do we deal with it

Reply:
As in previous cases, I'll revert back this change and add it in PR with verbose mode